src/remediations/fifi.js

```diff
@@ -242,16 +242,14 @@ function defineRHCEnabledExecutor (satellites, smart_management, rhc_enabled) {
     const satlessExecutor = _.find(satellites, satellite => satellite.id === null);
     if (satlessExecutor) {
         const partitionedSystems = sortRHCSystems(satlessExecutor, smart_management);
+        _.remove(satellites, satlessExecutor); // Remove redundant satless executor
```

Comment:
Based on https://lodash.com/docs/4.17.15#remove the function expects a predicate as the second parameter. Even if this works somehow it would be better to modify this so that we do not depend on undocumented code.